lib/check_for_oplog.js

```diff
@@ -74,10 +17,12 @@ OplogCheck.env = function () {
 };
 
 OplogCheck.disableOplog = function (cursorDescription) {
-  if (cursorDescription.options._disableOplog) {
+  const { options } = cursorDescription;
+
+  if (options._disableOplog || options.disableOplog) {
```

Comment:
In situations like this where we have to check for the same thing multiple ways, I like to leave a comment explaining what Meteor versions each one is for, or why we have to check multiple ways.